integrate-sphinx.md

```diff
@@ -0,0 +1,71 @@
+
+A few other notes:
+* We don't currently support Scroll, Mantle, or Zora, but we'd be happy to support them if you want. Generally, you can just ask us if you need us to support any EVM-compatible network.
+* The diff of this integration is [here](https://github.com/sam-goldman/quest-protocol/pull/1).
+
+If you have any questions, check out our [GitHub page](https://github.com/sphinx-labs/sphinx) or send us a message.
+
+## Deployment Instructions
+
+1. Clone this fork of your repo:
+```
+git clone git@github.com:sam-goldman/quest-protocol.git
+```
+
+2. 
+```
+cd quest-protocol
+```
+
+3. Update Foundry, then install packages:
+```
+foundryup && bun install
+```
+
+4. Sign up for Sphinx using this [invite link](https://www.sphinx.dev/signup?code=7df7c3d8-8e66-41a0-b6bc-04cf2021cb35).
+
+5. In Sphinx's website, go to "Options" -> "API Credentials". You'll need these credentials in the next couple of steps.
+
+6. Open `script/SphinxConfig.sol`. The `setUp` function contains your config options. Update the following fields:\
+    a. In `sphinxConfig.orgId`, add the Organization ID from Sphinx's website. This is a public field, so you don't need to keep it secret.\
+    b. In `sphinxConfig.owners`, add the addresses of the account that will own your project. (Specifically, it'll own the Gnosis Safe that executes your deployment).
+
+7. Create a `.env` file. Then, copy and paste the variables from `.env.example` to `.env` and fill them in. The `SPHINX_API_KEY` is in the Sphinx UI (under "Options" -> "API Credentials").
+
+8. You're done with the configuration steps! Run `forge build` to make sure your contracts can compile.
+
+9. Next, propose `ProxyAdmin.s.sol` on the networks in `sphinxConfig.testnets` (which is currently just Arbitrum Sepolia):
+```
+bun sphinx propose script/ProxyAdmin.s.sol --networks testnets
+```
+
+10. When the proposal is finished, go to the [Sphinx UI](https://sphinx.dev) to approve the deployment. After you approve it, you can monitor the deployment's status in the UI while it's executed.
+
+### Setup the `QuestFactory` with Sphinx (Optional)
+
+In this section, you'll run the `QuestFactoryDeploy` script from your local machine so that the `QuestFactory` has the same address as your previous deployments. I've made one modification to this script: at the end, it transfers ownership of the `QuestFactory` from the `ProxyAdmin` to your Gnosis Safe, which was deployed earlier in this guide.
+
+Run the `QuestFactoryDeploy` script:
+```
+forge script script/QuestFactory.s.sol:QuestFactoryDeploy --rpc-url arbitrum_sepolia --broadcast --verify -vvvv
+```
+
+After this script executes, you can upgrade and manage the `QuestFactory` with Sphinx. For example, to run the `QuestFactoryUpgrade` script, run the command:
+```
+bun sphinx propose script/QuestFactory.s.sol --target-contract QuestFactoryUpgrade --networks testnets
```

Comment:
I also got a warning about no contract artifact found. I believe, that's expected but just thought I'd call it out in case it's not. Might be nice to mention it here, but I don't think that's required.